frontend/src/components/Map/index.tsx

```diff
@@ -1,9 +1,100 @@
-import React from 'react';
-import { MapContainer, TileLayer } from 'react-leaflet';
+import React, { useState, useEffect } from 'react';
+import {
+  MapContainer,
+  Marker,
+  Popup,
+  TileLayer,
+  useMapEvents
+} from 'react-leaflet';
+import { v4 } from 'uuid';
+import 'leaflet/dist/leaflet.css';
+import { lightLocationType } from '../../constants/develop';
+import { fetchData } from '../../fetch/requests';
+
+type latlngType = {
+  lat: number;
+  lng: number;
+};
+
+type boundsType = {
+  _northEast: {
+    lat: number;
+    lng: number;
+  };
+  _southWest: {
+    lat: number;
+    lng: number;
+  };
+};
```

Comment:
move to _types.ts_ file and export types from that file.
Also, I don't understand why u use underscore with northEast and _southWest variables